doc/content/language.md

```diff
@@ -961,6 +961,31 @@ will print
 
 (note that the string is modified but not the fields `duration` and `bpm`).
 
+### Optional fields
+
+During the execution of your script, it can be useful to allow functions
+to receive records that may or may not have a specific field. This can
+be used, for instance, to model optional arguments.
+
+Liquidsoap supports that functionality using the `?.` syntax. Here's an example:
+
+```liquidsoap
+# This functions adds 1 to x unless options has a
+# add field in which case it adds this value
+def f(x, options) =
+  x + (options?.add ?? 1)
+end
```

Comment:
I am not so fond of this `.?` syntax and moreover it seems to me that providing a default value is more primitive than returning `null` if we have nothing: most of the time we want to provide a default value. I therefore propose  
to add a new syntax `x.field ??? value` which would be parsed as such and would be nicer.

```suggestion
def f(x, options) =
  x + (options.add ??? 1)
end
```

You can always implement your `x?.field` as `x.field???null()` if needed.

On the long run, I think we will be able to drop most `null` values and leave the `??` syntax for records with optional fields (we can leave this for after, I don't expect that the syntax will be widely used soon).